docs/Tutorials/modifyUserAuth.md

```diff
@@ -1,13 +1,50 @@
 --8<-- "References/abbreviations.md"
 
 
-As part of the Open AMT Cloud Toolkit reference implementation, MPS and the Kong service issue and authenticate a JSON Web Token (JWT) for user authentication. The default configuration offers authentication functionality, but it does not support many common configuration options, such as user groups. In a production environment, alternative authentication is available in 0Auth 2*, Lightweight Directory Access Protocol (LDAP), Kerberos*, etc.
-
-!!! Warning
-    In the current release, if you choose to modify the toolkit's default authentication, no keyboard, video and mouse (KVM) or serial over LAN (SOL) support will be available. 
+As part of the Open AMT Cloud Toolkit reference implementation, MPS and the Kong service issue and authenticate a JSON Web Token (JWT) for user authentication. The default configuration offers authentication functionality, but it does not support many common configuration options, such as user groups. In a production environment, alternative authentication is available in 0Auth 2*, Lightweight Directory Access Protocol (LDAP), Kerberos*, and more.
 
 The instructions below explain how to add an LDAP plugin to Kong.
 
+## User Authentication Flow
+
+The following diagrams help illustrate the typical user authentication flow. Learn more about the `authorize` API call in the [REST API Call Tutorial](./apiTutorial.md) or directly in our [API Documentation](../APIs/indexMPS.md).
+
+### REST API User Authentication Flow
+
+<figure class="figure-image">
+<img src="..\..\assets\images\UserAuth_API_Diagram.png" alt="Figure 1: User Authentication Flow for REST APIs">
+</figure>
+
+#### To authenticate for REST APIs:
+
+1. Call the MPS REST API `/api/v1/authorize` to receive an authentication token from the User Auth Service.
```

Comment:
Do we need to add a clarification here saying "if using the simple MPS auth service"